asterius/src/Asterius/Boot.hs

```diff
@@ -108,6 +108,8 @@ bootRTSCmm BootArgs {..} =
                          "rts",
                          "-dcmm-lint",
                          "-O2",
+                         "-DASTERIUS",
+                         "-optc=-DASTERIUS",
```

Comment:
Only adding `-DASTERIUS` as a ghc flag is enough for it to be passed to the underlying CPP program. Do you think the `-DASTERIUS` should be unconditionally flipped on for `ahc`? Then we won't need to add this flag to a lot of places (here, boot script, patched cabal files in the future, etc)